apps/enc.c

```diff
@@ -50,7 +50,8 @@ typedef enum OPTION_choice {
 
 const OPTIONS enc_options[] = {
     {"help", OPT_HELP, '-', "Display this summary"},
-    {"ciphers", OPT_LIST, '-', "List ciphers"},
+    {"ciphers", OPT_LIST, '-', "Alias of -list"},
```

Comment:
NIT: doesn’t one say „alias _for_ sth.“ ?

Reply:
yup

Reply:
This has been addressed.